addons/payment_stripe/models/payment.py

```diff
@@ -90,6 +114,8 @@ def _create_stripe_session(self, kwargs):
         if resp.get('payment_intent') and kwargs.get('client_reference_id'):
             tx = self.env['payment.transaction'].sudo().search([('reference', '=', kwargs['client_reference_id'])])
             tx.stripe_payment_intent = resp['payment_intent']
+        if 'id' not in resp and 'error' in resp:
+            _logger.error(resp['error']['message'])
```

Comment:
A possible error is that user forgot to set an email for the invoiced customer. We should bubble up such an error. But I'm not sure I can bubble up all errors from Stripe. Or if it's the best place to do that. What do you think @dbo-odoo ?

Reply:
i'm not strongly opiniated on this, tbh
yes, we could `raise UserError(resp['error'].get('message', '/'))`, which would then be displayed as the error message in the payment form. From my experience with the stripe api, it should not leak extremely sensitive information (e.g. api keys or whatever), but it could leak a problem with your config which should not be customer-facing...

regarding the email issue, it is *somewhat* mitigated by #51526

leave it like this for now